monitoring/monitorlib/auth.py

```diff
@@ -254,29 +253,46 @@ def issue_token(self, intended_audience: str, scopes: List[str]) -> str:
 
 
 class ClientIdClientSecret(AuthAdapter):
-  """Auth adapter that gets JWTs using a client ID and client secret."""
+  """Auth adapter that gets JWTs using a client ID and client secret. By default, this will send the request as JSON, you can use send_request_as_data flag to send the request as form data. """
 
-  def __init__(self, token_endpoint: str, client_id: str, client_secret: str):
+  def __init__(self, token_endpoint: str, client_id: str, client_secret: str, send_request_as_data :bool=False):
```

Comment:
Constructor arguments [may only be strings](https://github.com/interuss/dss/blame/master/monitoring/monitorlib/README.md#L19) and bool isn't a string.  To accept `send_request_as_data` as a `str`, convert to a bool with something like `send_request_as_data.lower() == 'true'`.

Reply:
Thanks, fixed this now.